src/user/user.service.ts

```diff
@@ -30,6 +25,11 @@ export class UserService {
     return user;
   }
 
+  async create(createUserDto: CreateUserDto) {
+    const user = await new this.userModel(createUserDto);
+    return user.save();
+  }
+
```

Comment:
widzę, że miała być kolejność taka jak w pass.service ;D tyle, że to nie jest powiązane z danym pull requestem i trzeba być ostrożnym w zmianach na kodzie który jest zmergowany i na którym już pracują inni, chociaż tu nie powinno być problemów, jeśli nikt inny nic tam nie działał ;D

Reply:
pozwoliłem sobie zmienić jako że to też moj task ;)

Reply:
ale dobra tez zmienie xD

Reply:
no juz zostaw xd to na przyszłość bardziej :D